.gitpod.yml

```diff
@@ -0,0 +1,7 @@
+tasks:
+  - init: yarn install && yarn build
+    command: yarn develop
+
+ports:
+  - port: 8000
+    onOpen: open-preview
```

Comment:
does this file have to be in the repo root? Couldn't find anything in https://www.gitpod.io/docs/config-gitpod-file/ about location

Reply:
Yeah, it has to be in the repo root. Sorry! that It's not documented.

Reply:
👍